packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/empty-state/empty-state.md

```diff
@@ -5,64 +5,186 @@ hideTOC: true
+
+1. **Call-to-action as a primary button**
+![Primary button one cta](./img/one-ctas-primary.jpg) 
+
+2. **Call-to-action in body**
+![In body cta](./img/one-cta-body.jpg)
+
+If you have more than one actionable link, there are two options depending on how your primary call-to-action is presented:
+1. **Link button underneath the primary button:** if your primary call-to-action was in a primary button format
+
+![Primary button two ctas](./img/two-ctas-primary.jpg) 
+
+2. **Link button underneath the body:** if your first call to action was in the body.
+
+![In body two ctas](./img/two-ctas-body.jpg)
+
+There are only two situations where a stand-alone link button (with nothing linked in the body) can be used:
+1. **Filtering or search situation:** when a user has filtered for something in a table, and is prompted to "clear all filters" or "include" other items in their search
+2. **Multiple primary calls-to-action:** when there are multiple primary actions the user can take and they are all equally important
+
+![Solo link cta](./img/solo-link.jpg)
+
+### Alignment
+
+**Full-page empty state**
+
+When an empty state appears on a full-page,  it  should be 64px away from the top, and horizontally centered with a minimum padding of 24px on all remaining sides.
+
+![Full-page empty state](./img/desktop-page-es.jpg)
+
+Mobile in-page empty states should be 48px away from the top and have a minimum of 16px padding on the remaining sides.
+
+![Mobile page empty state](./img/mobile-page-es.jpg)
+
+**In-table, card or wizard empty states**
+
+Empty states within a table, card or wizard should always be horizontally and vertically centered on their background, with an outside padding of 24px on all sides.
+
+
+![In table empty state](./img/table-es.jpg)
+
+### Toolbars or filters in empty states
+
+No toolbars or filters should be shown when a user is in a full-page empty state as the user will have no items to perform any actions on.
+
+![Toolbar usage](./img/toolbar.jpg)
 
-1. **Icon/image (optional):** Display associated object icons here. An icon is only recommended for full page empty states
-2. **Title:** Provide a concise explanation
-3. **Body:** Provide additional information that helps a user understand why the space is empty, what they can do to move forward, and the value or benefit that is gained by taking the next step
-4. **Primary button:** Primary call to action
-5. **Secondary buttons (optional):**  Alternative options for the user. There can be more than one secondary action.
 
 ## Usage
 There are several common use cases for empty states:
+* [First use or get started](#first-use-or-get-started)
+* [No data to show](#no-data-to-show)
+* [Required configuration](#required-configuration)
```

Comment:
This link is not working. I think it's because it needs to match the actual heading text

Reply:
Edited heading to match the link - should work now!